config/requests.yml

```diff
@@ -13,6 +13,8 @@ defaults: &defaults
 development:
   <<: *defaults
   voyager_ub_id: 1@DB
+  bibdata_base: https://bibdata-dev.princeton.edu
+  stackmap_base: https://bibdata-dev.princeton.edu/locations/map
```

Comment:
@hackmastera I don't think we need the stackmap_base key set any more. That one was purged when we took that code out of bibdata last week.

Reply:
oh, great. thanks will take care of it.